ztools/x509/json.go

```diff
@@ -322,3 +337,18 @@ func (c *Certificate) MarshalJSON() ([]byte, error) {
 
 	return json.Marshal(jc)
 }
+
+func purgeNameDuplicates(names []string) (out []string){
```

Comment:
That's roughly how this already works, except that it is using a map[string]bool instead.